library/src/main/java/com/yuyh/library/EasyGuide.java

```diff
@@ -1,384 +1,416 @@
+        tipView.setAlignment(LayoutAlignment.HORIZONTAL_CENTER);
+        tipView.setLayoutConfig(new DirectionalLayout.LayoutConfig(
+                ComponentContainer.LayoutConfig.MATCH_PARENT, ComponentContainer.LayoutConfig.MATCH_CONTENT));
+        tipView.setOrientation(DirectionalLayout.VERTICAL);
         if (mIndicators != null) {
             for (TipsView tipsView : mIndicators) {
-                addView(tipsView.view, tipsView.offsetX, tipsView.offsetY, tipsView.params);
+                addView(tipsView.getView(), tipsView.getOffsetX(), tipsView.getOffsetY(), tipsView.getParams());
             }
         }
-
         if (mMessages != null) {
-            int padding = dip2px(mActivity, 5);
+            int padding = dip2px(mAbilitySlice, 5);
             for (Message message : mMessages) {
-                TextView tvMsg = new TextView(mActivity);
-                tvMsg.setLayoutParams(new ViewGroup.LayoutParams(MATCH_PARENT, WRAP_CONTENT));
+                Text tvMsg = new Text(mAbilitySlice);
+                tvMsg.setLayoutConfig(new ComponentContainer.LayoutConfig(
+                        ComponentContainer.LayoutConfig.MATCH_PARENT, ComponentContainer.LayoutConfig.MATCH_CONTENT));
                 tvMsg.setPadding(padding, padding, padding, padding);
-                tvMsg.setGravity(Gravity.CENTER);
-                tvMsg.setText(message.message);
-                tvMsg.setTextColor(Color.WHITE);
-                tvMsg.setTextSize(message.textSize == -1 ? 12 : message.textSize);
-
-                mTipView.addView(tvMsg);
+                tvMsg.setTextAlignment(TextAlignment.CENTER);
+                tvMsg.setText(message.getMessagetext());
+                Color hmosColor = EasyGuide.changeParamToColor(ohos.agp.utils.Color.WHITE.getValue());
+                tvMsg.setTextColor(hmosColor);
+                tvMsg.setTextSize(50);
+                tipView.addComponent(tvMsg);
             }
         }
-
         if (mConfirm != null) {
-            TextView tvConfirm = new TextView(mActivity);
-            tvConfirm.setGravity(Gravity.CENTER);
-            tvConfirm.setText(mConfirm.text);
-            tvConfirm.setTextColor(Color.WHITE);
-            tvConfirm.setTextSize(mConfirm.textSize == -1 ? 13 : mConfirm.textSize);
-            tvConfirm.setBackgroundResource(R.drawable.btn_selector);
-            LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(WRAP_CONTENT, WRAP_CONTENT);
-            params.topMargin = dip2px(mActivity, 10);
-            tvConfirm.setLayoutParams(params);
-            int lr = dip2px(mActivity, 8);
-            int tb = dip2px(mActivity, 5);
+            Text tvConfirm = new Text(mAbilitySlice);
+            tvConfirm.setTextAlignment(TextAlignment.CENTER);
+            tvConfirm.setText(mConfirm.getText());
+            Color hmosColor1 = EasyGuide.changeParamToColor(ohos.agp.utils.Color.WHITE.getValue());
+            tvConfirm.setTextColor(hmosColor1);
```

Comment:
setTextSize must not be hardcode value. It should depend on mConfim.textSize value